bin/serve

```diff
@@ -89,13 +90,13 @@ tfe.set_tfe_trace_flag(enable_traces)
 tfe.set_tfe_debug_flag(enable_debug)
 
 remote_config = tfe.RemoteConfig(
-    {
-        'master': master_address,
-        'server0': server0_address,
-        'server1': server1_address,
-        'crypto-producer': producer_address,
-        'weights-provider': provider_address
-    }
+    OrderedDict([
```

Comment:
👍 you could probably also use 
```python
OrderedDict(
  master=...,
  server0=..., )
```

Reply:
Much nicer! Thanks.

Reply:
although its doesn't like names like `crypto-producer` in the argument list. Will leave it how it is for now.

Reply:
Doens't like the `-` character.